Third third of corrections of typos, grammar, and readability (#39)
mia-entropy/eberban: books/refgram/src/from_scratch/core_to_complete/pairs.md

# Pairs

An important building block is the ability to construct more complex objects
from simple ones. For this reason, we define the concept of a pair, which is an
atom that is associated with a left and right component.

> Anything in Eberban is a component. We'll use an axiom to state that there

Reviewer: > Anything in Eberban is a component.
No idea what that mean. However anything can fit inside the left or right component of pair.

> An important building block is the ability to construct more complex objects
> from simple ones. For this reason, we define the concept of a pair, which is an
> atom that is associated with a left and right component.
> 
> Anything in Eberban can go inside a component. We'll use an axiom to state that
> there exists a unique atom for every possible pair of left and right components.

Reviewer: ```suggestion
An important building block is the ability to construct more complex objects
from simple ones. For this reason, we define the concept of a pair, which is an
atom that is associated with a left and right component. Anything (atom, predicate) can be the left or right component of a pair.

We'll use an axiom to state that there exists a unique atom for every possible pair of left and right components.
```
